zulipterminal/ui_tools/boxes.py

```diff
@@ -1079,11 +1080,13 @@ def footlinks_view(
     def soup2markup(
         cls, soup: Any, metadata: Dict[str, Any], **state: Any
     ) -> Tuple[
-        List[Any], "OrderedDict[str, Tuple[str, int, bool]]", List[Tuple[str, str]]
+        List[urwidTextMarkup],
```

Comment:
Do we need to wrap `urwidTextMarkup` inside a `List`?
`urwidTextMarkup` already contains an outer layer of `Sequence` type which should cover `List`?

Reply:
I'm skeptical about this too. I think `Sequence` might only be applicable as a type if the contents in the sequence are not changing? (I think I discussed this on stream too?)